radius.go

```diff
@@ -12,6 +12,9 @@ import (
 
 )
 
+// bugs: placeholders for the sql lib
```

Comment:
I'm confused by what this commit means? I haven't seen this pattern before. Is the intention to turn this into a github issue once the PR is accepted?